### ISSUE TITLE
Nuxt 0.1.0 issues with d.ts files

### ISSUE THREAD

**Issue by husayt** [prisma/nuxt-prisma#61]
Getting this error after upgrading to 0.1.0
```
ERROR  RollupError: ../node_modules/@prisma/nuxt/dist/runtime/server/utils/prisma.d.ts (2:8): Expected ';', '}' or <eof> (Note that you need plugins to import files that are not JavaScript)


import { PrismaClient } from "@prisma/client";
declare const prismaClientSingleton: () => PrismaClient<import(".prisma/client").Prisma.PrismaClientOptions, never, i...
           ^
declare const prisma: PrismaClient<import(".prisma/client").Prisma.PrismaClientOptions, never, import("@prisma/client...
export type CustomPrismaClient = ReturnType<typeof prismaClientSingleton>;
```

**Comment by ankur-arch** [comment]
Thanks for reporting @husayt! I'll look into it!

**Comment by ankur-arch** [comment]
Hi @husayt, I appreciate your patience and apologize for the long wait.  

I’ve picked this up again and wanted to clarify: when exactly does this issue occur? Also, which package manager are you using?

**Comment by ul8** [comment]
@ankur-arch I can jump in here since I have the same issue. I scaffolded a nuxt project using the latest versions a few days ago, see below my deps (removed nuxt-prisma in the meantime but also used 0.1.0)

I'm using pnpm and this occurs when I run `pnpm run dev`

```
 "dependencies": {
    "@nuxt/eslint": "^0.7.1",
    "@nuxt/eslint-config": "^0.7.1",
    "@nuxtjs/i18n": "^9.1.0",
    "@nuxtjs/seo": "^2.0.0-rc.23",
    "@nuxtjs/tailwindcss": "^6.12.2",
    "@pinia/nuxt": "^0.7.0",
    "@vueuse/core": "^11.3.0",
    "@vueuse/nuxt": "^11.3.0",
    "i18n": "^0.15.1",
    "nuxt": "^3.14.1592",
    "vue": "latest",
    "vue-router": "latest"
  },
  ```

**Comment by ankur-arch** [comment]
Thanks so much for your input on this issue 🙏,

@husayt, I’ve addressed some of your feedback in this [comment](https://github.com/prisma/nuxt-prisma/issues/63#issuecomment-2504023628) — I appreciate your patience as I continue working through this.

Thanks @ul8! I’ll review this later this week and share any updates or findings.

**Comment by MuhamadAdrian** [comment]
In my case, the error comes when using sidebase/nuxt-auth and prisma/nuxt in the modules. But if im commented out the sidebase/nuxt-auth plugin, there is no error.

**Comment by gsxdsm** [comment]
Same issue. Using Nuxt/ui also breaks now with prisma

**Comment by koalli** [comment]
Same Issue. Both [side/nuxt-auth](https://github.com/sidebase/nuxt-auth) and  [atinux/nuxt-auth-utils](https://github.com/atinux/nuxt-auth-utils) will cause this problem. Just add these modules to the project and `npm run dev`.

**Comment by ankur-arch** [comment]
> For me it was now fixed by simply installing the prisma and @prisma/client packages, and creating a prismaClient instance manually.

I couldn't reproduce the exact error, but this resolves the issue and allows the project to run. It seems npm is failing to properly install and set up Prisma, leading to a preview failure with this error. Following the [advice](https://www.prisma.io/docs/orm/more/help-and-troubleshooting/help-articles/prisma-nuxt-module#resolving-typeerror-failed-to-resolve-module-specifier-prismaclientindex-browser) in the documentation helped resolve it.

> Same Issue. Both [[side/nuxt-auth](https://github.com/sidebase/nuxt-auth)](https://github.com/sidebase/nuxt-auth) and [[atinux/nuxt-auth-utils](https://github.com/atinux/nuxt-auth-utils)](https://github.com/atinux/nuxt-auth-utils) will cause this problem. Just add these modules to the project and npm run dev.

Thanks, @koalli! I'll try this out and see if I can reproduce the exact error. I'm a bit tied up until tomorrow but will aim to get to this by Friday. Thanks, everyone!

**Comment by gsxdsm** [comment]
Npm is installing and setting up prisma just fine. In fact you can prove this by deleting the .nuxt folder and doing a build. It will work perfectly. The first time. Build again and it fails. And the article you linked to with a fix doesn't work and is unrelated to this. There's a problem with how rollup is processing prisma and specifically the typescript portion. Something in prisma nuxt is not cooperating with rollup. I've been trying to debug the build step in Nuxt but haven't pinpointed it yet.

**Comment by ankur-arch** [comment]
Thanks for the insight, @gsxdsm!

> The article you linked to with a fix doesn't work and is unrelated to this.

I haven't been able to reproduce the exact error yet, as I haven’t tried @koalli's suggestion. In my case, I encountered the preview error linked here, which is why I mentioned it.

> There's a problem with how rollup is processing Prisma, specifically the TypeScript portion. Something in Prisma Nuxt is not cooperating with Rollup. I've been trying to debug the Nuxt build step but haven't pinpointed it yet.

If you have any further insights or updates, I’d greatly appreciate it 🫶 !

**Comment by ankur-arch** [comment]
Hey there @koalli, thanks so much for your suggestion and for your patience!

Could you please check if this issue is resolved in version 0.1.3? It appears to be working fine on my end. 😊

**Comment by koalli** [comment]
@ankur-arch I tested version 0.1.3 and added the `nuxi-auth-utils` module to my previous project, everything seems fine now, and it no longer throws a RollupError.

**Comment by ankur-arch** [comment]
Thank you, @koalli for getting back to me and @husayt for reporting the issue.

This [PR #68](https://github.com/prisma/nuxt-prisma/pull/68/files) resolves the problem and has been included in version 0.1.3.
